api.go

```diff
@@ -167,8 +169,20 @@ type (
 		CreatedAt     int64  `json:"createdAt"`
 		InputAssetID  string `json:"inputAssetId,omitempty"`
 		OutputAssetID string `json:"outputAssetId,omitempty"`
-		Type          string `json:"type"`
-		Params        struct {
+		Output        struct {
```

Comment:
**OCD-level-nit:** WDYT of moving this to after the `type` and `params` fields instead, since they are something that only show up later in a task lifecycle? First it is created only with a type and params, only later the output is populated when it finishes.

Totally optional.